.jenkins

```diff
@@ -219,7 +219,7 @@ pipeline {
                 recordIssues(
                     enabledForFailure: true,
                     tools: [cmake(), gcc(), clang(), clangTidy()],
-                    qualityGates: [[threshold: 1, type: 'TOTAL', unstable: true]],
+                    qualityGates: [[threshold: 0, type: 'TOTAL', unstable: true]],
```

Comment:
This will fail all the time

Reply:
I noticed. I looked at the documentation, but it does is incomprehensible to me. Is commenting this line the right solution?